build/IPDK_Container/Ubuntu20.04/Makefile

```diff
@@ -0,0 +1,48 @@
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+#
+# When using sample configurations from 'examples' folder, do the following to get your P4-OVS container up and running:
+# make build
+# make volume
+#
+
+# Import variables needed by this Makefile
+env_var ?= docker.env
+include $(env_var)
+export $(shell sed 's/=.*//' $(env_var))
+
+.PHONY: help
+
+## List all available commands
+help:;@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+#awk '/^#/{c=substr($$0,3);next}c&&/^[[:alpha:]][[:alnum:]_-]+:/{print substr($$1,1,index($$1,":")),c}1{c=0}' $(MAKEFILE_LIST) | column -s: -t
+
+# DOCKER COMMANDS
+## Build the docker image
+build: ;docker build -t ${IMAGE_NAME}:${TAG} -f Dockerfile . \
+--build-arg http_proxy=${PROXY} \
+--build-arg https_proxy=${PROXY} \
+--build-arg OS_VERSION=${OS_VERSION}
```

Comment:
Would it be possible to add a build Makefile target without http_proxy/https_proxy?

Reply:
I have added 2 more targets. build-proxy and build-nc-proxy. 
so use build and build-nc if you are not behind proxy.
And use build-proxy and build-nc-proxy if you are behind proxy.